src/status_im/ui/screens/wallet/views.cljs

```diff
@@ -36,17 +37,15 @@
      [react/text {:style               styles/total-balance-currency
                   :accessibility-label :total-amount-currency-text}
       (:code currency)]]
-    [react/text {:style styles/total-value} (i18n/label :t/wallet-total-value)]]])
+    [l/label styles/total-value :wallet-total-value]]])
```

Comment:
I noticed all i18n keywords are de-qualified. Why is that?

Reply:
it was my initial idea, what's the reason to qualify them? it's difficult to search translation

Reply:
fair enough